libraries/chain/fork_database.cpp

```diff
@@ -331,15 +332,22 @@ namespace eosio::chain {
       EOS_ASSERT( prev_bh, unlinkable_block_exception,
                   "unlinkable block", ("id", n->id())("previous", n->previous()) );
 
-      if( validate ) {
+      if (validate) {
          try {
             const auto& exts = n->header_exts;
 
-            if( exts.count(protocol_feature_activation::extension_id()) > 0 ) {
-               const auto& new_protocol_features = std::get<protocol_feature_activation>(exts.lower_bound(protocol_feature_activation::extension_id())->second).protocol_features;
-               validator( n->timestamp(), static_cast<bs*>(prev_bh.get())->get_activated_protocol_features()->protocol_features, new_protocol_features );
+            if (exts.count(protocol_feature_activation::extension_id()) > 0) {
+               const auto& new_protocol_features =
+                  std::get<protocol_feature_activation>(
+                     exts.lower_bound(protocol_feature_activation::extension_id())->second)
+                     .protocol_features;
+               validator(n->timestamp(),
+                         static_cast<bs*>(prev_bh.get())->get_activated_protocol_features()->protocol_features,
+                         new_protocol_features);
```

Comment:
I can see the original being too long, but this is too short and kind of a mess.

Reply:
Right, fixed it!